src/cs-core/tools/MultiPointTool.cpp

```diff
@@ -15,13 +15,12 @@ namespace cs::core::tools {
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 MultiPointTool::MultiPointTool(std::shared_ptr<InputManager> const& pInputManager,
-    std::shared_ptr<SolarSystem> const&                             pSolarSystem,
-    std::shared_ptr<GraphicsEngine> const&                          graphicsEngine,
+    std::shared_ptr<SolarSystem> const& pSolarSystem, std::shared_ptr<Settings> const& settings,
     std::shared_ptr<TimeControl> const& pTimeControl, std::string const& sCenter,
     std::string const& sFrame)
     : mInputManager(pInputManager)
     , mSolarSystem(pSolarSystem)
-    , mGraphicsEngine(graphicsEngine)
+    , mSettings(settings)
     , mTimeControl(pTimeControl)
     , mCenter(sCenter)
     , mFrame(sFrame) {
```

Comment:
Pass by value and move.

Reply:
You're right but in order to avoid too many conflicts with the `feature/clang-tidy` I will do this after it got merged.